StandardLibraryEssentials/StandardTemplateLibrary/STLAlgorithms/task.md

```diff
@@ -0,0 +1,179 @@
+In addition to all that we have learned so far, the STL also provides a number of algorithms that can be used on a range of elements. These algorithms are implemented as functions that usually utilize iterators. In this task, we will look at some of the most commonly used algorithms. There are many more algorithms available in the STL, and you can find a complete list [here](https://en.cppreference.com/w/cpp/algorithm).
+We highly encourage you to explore the STL algorithms by yourself and try to use them in your code! They are useful and can save you a lot of time and effort.
+
+#### 1. `std::sort()`
```

Comment:
Please mention iterators of which category each algorithm expects.

Reply:
This comment is still relevant (there is still no mentioned of iterators categories expected by the algorithms).